### PR TITLE
update linter to remove Google Cloud and ingest suggestions

### PR DESCRIPTION
**Pull request by jacobprall**
# Description

[Short summary of why you created this PR]

# Version

Which documentation version does this PR apply to?

- [ ] Latest (Default)
- [ ] Version 1.7
- [ ] Older [specify]

# Links

Fixes #[insert issue link, if any]

### CODE REVIEW

**Review by Loquacity — APPROVED** [review]

**Comment by Loquacity** [comment]
@jacobprall I've noticed we typo Timescale as "Timeascale" (and variations) a lot, I wonder if we can add a rule for that?

File: .github/styles/Google/WordList.yml [hunk 1]
```diff
@@ -58,7 +57,6 @@ swap:
   grayed-out: unavailable
   HTTPs: HTTPS
   in order to: to
```
Comment (charislam) [inline .github/styles/Google/WordList.yml:59]:
```suggestion
  in order to: to
  indices: indexes
```

Comment (charislam) [reply .github/styles/Google/WordList.yml:59]: Noticed this one popped up sometimes when I was reading through the docs.
Also `Postgres: PostgreSQL`?

Comment (Loquacity) [reply .github/styles/Google/WordList.yml:59]: Yes, 'in order to' happens a lot 😉  And yes, Postgres should always be PostgreSQL 👍

**Pull request merged**